h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -59,18 +64,39 @@ public RapidsSchemaV3 exec(int version, RapidsSchemaV3 rapids) {
     }
   }
 
+  private static Class[] getRapidsClasses(){
```

Comment:
The rapids help prints also abstract rapids classes. The abstract ones are for example `AstRoot`, `AstBuiltin` and way more. But I think it makes sense to not include the abstract ones into the help output